### PR TITLE
Ion mobility filtering for full scan data should use high energy ion mobility offset values for all acquisition types

### PR DESCRIPTION
**Pull request by bspratt**
Formerly we applied it only in All Ions mode, but it is also desirable in PRM etc.

Reported by user Anne

### CODE REVIEW

File: pwiz_tools/Skyline/TestPerf/PerfHighEnergyIonMobilityOffsetForPRM.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        public void TestHighEnergyIonMobilityOffsetForPRM()
+        {
+
+            TestFilesZip = GetPerfTestDataURL(@"PerfTestPrmDtOffset.zip");
+
+            RunFunctionalTest();
+            
+        }
+
+        private string GetTestPath(string relativePath)
```
Comment (nickshulman) [inline pwiz_tools/Skyline/TestPerf/PerfHighEnergyIonMobilityOffsetForPRM.cs:45]: This function doesn't seem to be used.

**Review by nickshulman — APPROVED** [review]
Looks good to me.

File: pwiz_tools/Skyline/TestPerf/PerfHighEnergyIonMobilityOffsetForPRM.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        protected override void DoTest()
+        {
+            string skyfile = TestFilesDir.GetTestPath("prm_dt_offset.sky");
+
+            RunUI(() => SkylineWindow.OpenFile(skyfile));
+
+            var doc = WaitForDocumentLoaded();
+            AssertEx.IsDocumentState(doc, null, 1, 1, 1, 2);
+            ImportResults("data.mzML");
+            doc = WaitForDocumentChangeLoaded(doc);
```
[the remark this answers is not in the document]
Comment (bspratt) [reply pwiz_tools/Skyline/TestPerf/PerfHighEnergyIonMobilityOffsetForPRM.cs:60]: I just need to get the state of the document after it's loaded - I guess SkylineWindow.Document would work just as well. But this seems OK to me?

File: pwiz_tools/Skyline/TestPerf/PerfHighEnergyIonMobilityOffsetForPRM.cs (hunk 1 above)
Comment (bspratt) [reply pwiz_tools/Skyline/TestPerf/PerfHighEnergyIonMobilityOffsetForPRM.cs:45]: Oops, yeah, a little cruft from a copy and paste, thanks

**Pull request merged**